pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -950,7 +950,7 @@ cdef class _Timedelta(timedelta):
         return np.int64(self.value).view('m8[ns]')
 
     @property
-    def resolution(self):
+    def reso_str(self):
```

Comment:
where did this name come from? maybe min_resolution_string? or just resolution_string is more informative

Reply:
More or less chosen out of thin air.  I think we might use it in some of the internal functions.  resolution_string sounds better, will update